src/pages/ResetPassword.tsx

```diff
@@ -100,9 +100,10 @@ export const ResetPassword = () => {
               <ul>
                 <li>at least 12 characters long,</li>
                 <li>
-                  a combination of uppercase letters, lowercase letters,
-                  numbers, and symbols.
+                  a combination of uppercase letters, lowercase letters and
+                  numbers
                 </li>
+                <li>at least one symbol from: <span style={{fontFamily: 'monospace'}}>! @ # $ % ^ &amp; *</span></li>
```

Comment:
@quietbits I'm not really sure what's the best way to format this message. If you have any suggestions please let me know. 

We just want to make sure that people know it's one of these symbols that count, and something like `-` doesn't.

Reply:
I think this is fine. At least it's very clear what is required. I would just apply the style through class instead of inline (comment below). Otherwise, this seems good to me. :+1: